### ISSUE TITLE
RAGAN-LS loss is difference between code and paper

### ISSUE THREAD

**Issue by xhhong** [TAMU-VITA/DeblurGANv2#10]
Hi~
Thanks for your work. 
There are some question about RAGAN-LS loss, the implementation in code is
 self.loss_D = (torch.mean((self.pred_real - torch.mean(self.fake_pool.query()) - 1) ** 2) +
                      torch.mean((self.pred_fake - torch.mean(self.real_pool.query()) + 1) ** 2)) / 2

but the details in paper is
LRaLSGAN
D = Ex∼pdata(x)(D(x) − Ez∼pz(z)D(G(z)) − 1)2
      + Ez∼pz(z)(D(G(z)) − Ex∼pdata(x)D(x) + 1)2

I think its difference between  "torch.mean(self.fake_pool.query())" and Ez∼pz(z)D(G(z)).

**Comment by Sandbox3aster** [comment]
Hi Hong,

After carefully check the code, i found it the implementation consistent with the paper,
In the paper, RaLSGAN Loss is described as:
![image](https://user-images.githubusercontent.com/6824524/87861858-0ba14700-c910-11ea-8f32-586bb1098e17.png)
In the repo, we use:
`self.loss_D = (torch.mean((self.pred_real - torch.mean(self.fake_pool.query()) - 1) ** 2) +
                       torch.mean((self.pred_fake - torch.mean(self.real_pool.query()) + 1) ** 2)) / 2`
only objectives different by a multiple of a constant (1/2)

Best,
Junru
